nox/_options.py

```diff
@@ -58,11 +74,33 @@ def _sessions_default():
 
 
 def _color_finalizer(value, args):
-    """Figures out the correct value for "color" based on the two color flags."""
-    return not args.nocolor or args.forcecolor
+    """Figures out the correct value for "color" based on the two color flags.
 
+    Args:
+        value (bool): The current value of the "color" option.
+        args (_option_set.Namespace): The values for all options.
 
-def _posargs_finalizer(value, args):
+    Returns:
+        The new value for the "color" option.
+    """
+    if args.forcecolor is True and args.nocolor is True:
```

Comment:
Here you use an `is True` check but on 91 and 94 you are just doing a truthy check. You should do the same thing in both places.

Reply:
Fair enough. Done.